### PR TITLE
[circle-mpqsolver] Introduce setVisqPath

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit introduce setVisqPath into BisectionSolver.

Draft: https://github.com/Samsung/ONE/pull/10432
Related: https://github.com/Samsung/ONE/issues/10393

ONE-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

**Review by jinevening — DISMISSED** [review]
LGTM

File: compiler/circle-mpqsolver/src/bisection/BisectionSolver.cpp [hunk 1]
```diff
@@ -101,6 +105,8 @@ float BisectionSolver::evaluate(const DatasetEvaluator &evaluator, const std::st
 
 void BisectionSolver::algorithm(Algorithm algorithm) { _algorithm = algorithm; }
 
+void BisectionSolver::setVisqPath(const std::string &visq_path) { _visq_data_path = visq_path; }
```
Comment (seanshpark) [inline compiler/circle-mpqsolver/src/bisection/BisectionSolver.cpp:108]: this doesn't seem to be related with replacing

**Review by jinevening — APPROVED** [review]

**Comment by seanshpark** [comment]
@nnfw-bot test nncc-debug
